serverless/networking/serverless-ossm.adoc

```diff
@@ -23,12 +23,14 @@ These options include setting custom domains, using TLS certificates, and using
 apiVersion: maistra.io/v1
 kind: ServiceMeshMemberRoll
 metadata:
-  name: default
+  name: default <1>
   namespace: istio-system
 spec:
   members:
-    - default
+    - default <2>
 ----
+<1> A name of {ProductShortName} Member Roll object, that isn't connected to namespace from <2>
+<2> A list of namespaces you like to integrate {ServerlessProductName} with {ProductName}
```

Comment:
```suggestion
<2> A list of namespaces where {ServerlessProductName} will be integrated with {ProductName}.
```

Reply:
+1
I think this is one of the rare places where using future tense makes the meaning clearer.